inbm/dispatcher-agent/requirements.txt

```diff
@@ -11,4 +11,5 @@ urllib3==1.26.18
 certifi==2023.7.22
 pyinstaller==5.13.1
 setuptools==65.5.1
-future==0.18.3
\ No newline at end of file
```

Comment:
Ensure there is a newline at the end of the `requirements.txt` file to conform to POSIX standards. [medium]